runtime/vm/j9vm.tdf

```diff
@@ -779,3 +779,6 @@ TraceExit=Trc_VM_sendResolveConstantDynamic_Exit Overhead=1 Level=2 Template="se
 TraceException=Trc_VM_CreateRAMClassFromROMClass_nestedValueClassNotVisible Overhead=1 Level=1 Template="Nested field (RAM class=%p, classloader=%p, this classloader=%p) is not visible. Throw IllegalAccessError"
 
 TraceEvent=Trc_VM_CreateRAMClassFromROMClass_valueTypeIsFlattened Overhead=1 Level=7 Template="ValueType is eligible for flattening name=%.*s j9class=%p"
+
+TraceEvent=Trc_VM_SetSoftFileLimit Overhead=1 Level=3 Template="Set max number of files to %zu"
```

Comment:
`%zu` is not correct for uint64_t as this expect 32-bit for a 32-bit JVM, I think it's `%llu`